aider.el

```diff
@@ -2,7 +2,7 @@
 
 ;; Author: Kang Tu <tninja@gmail.com>
 ;; Version: 0.2.0
-;; Package-Requires: ((emacs "26.1") (transient "0.3.0"))
+;; Package-Requires: ((emacs "26.1") (transient "0.3.0") (magit "4.0.0"))
```

Comment:
Thanks for updating. Just checked the magit repository, and seems the magit-toplevel function was available since 2.1.0. Wonder if we can make the magit level lower to v2? v4 is a pretty new version, and I doubt that lots of people actually didn't upgrade to that version yet.

Reply:
turns out i can change it directly.. just updated